textblob/_text.py

```diff
@@ -215,8 +215,8 @@ def penntreebank2universal(token, tag):
     "pred.", "pres.", "p.m.", "ref.", "v.", "vs.", "w/"
 ))
 
-RE_ABBR1 = re.compile("^[A-Za-z]\.$")       # single letter, "T. De Smedt"
-RE_ABBR2 = re.compile("^([A-Za-z]\.)+$")    # alternating letters, "U.S."
+RE_ABBR1 = re.compile("^[A-Za-z][.]$")      # single letter, "T. De Smedt"
```

Comment:
Instead of adding character classes, you can just turn all this into raw string which is de-facto standard for regex: `r"^[A-Za-z]\.$"` - such solution will be more understandable for almost everybody.

Reply:
I don't care what goes in, really. I would just like the syntax warning eliminated. A character class for a period is my preferred way to do it, but if raw strings and backslashes are more acceptable, that's cool too.

Reply:
Yep, I fully agree that we need this ASAP. I'll open another PR with raw string approach tonight and hope that maintainer will choose one of them and make a release. Unfortunately, seems like `textblob` is not maintained any longer, last commit to master was more than half a year ago.

Reply:
Any decision on this or @sterliakov's PR?